types/brotli/compress.d.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="node" />
+
+interface CompressOptions {
```

Comment:
```suggestion
interface CompressOptions {
```
same, simpler to maintain, I'll add suggestion to `index.d.ts`